### PR TITLE
requesting angular-rickshaw library on CDN

### PR DESCRIPTION
**Pull request by girishakj**

### CODE REVIEW

File: files/angular-rickshaw/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Nick Ng"
+github = "https://github.com/ngyewch"
```
Comment (jimaek) [inline files/angular-rickshaw/info.ini]: Must link to the project not the author

File: files/angular-rickshaw/update.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "packageManager": "github",
+  "name": "angular-rickshaw",
+  "repo": "ngyewch/angular-rickshaw",
+  "files": {
+    "basePath": "/",
+    "include": ["README.MD", "LICENSE.html","bower.json","rickshaw.js"],
```
Comment (jimaek) [inline files/angular-rickshaw/update.json]: None of these files is needed in production. Can you add a minified version to your repo?

File: files/angular-rickshaw/update.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    "exclude": [".gitignore","CHANGELOG.MD"]
```
Comment (jimaek) [inline files/angular-rickshaw/update.json]: No need for exclude when using include at the same time

**Comment by girishakj** [comment]
Closing this request & re-submitting with suggested changes.

**Pull request closed without merging**